styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```

Comment:
Please make sure that you are using the class with styles for your header instead of targeting it directly over the tag name.

Reply:
Done 🥇